Resources/Prototypes/Entities/Objects/Misc/spider_web.yml

```
@@ -110,6 +110,9 @@
     - type: Slippery
     - type: StepTrigger
       intersectRatio: 0.2
+      triggerGroups: # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)